src/main/java/org/codeforamerica/shiba/output/FileDownloadController.java

```diff
@@ -67,6 +69,55 @@ public FileDownloadController(
     ApplicationFile applicationFile = pdfGenerator.generate(applicationData.getId(), CAF, CLIENT);
     return createResponse(applicationFile);
   }
+  
+  @GetMapping("/download/{applicationId}")
+  ResponseEntity<byte[]> downloadAllDocumentsWithApplicationId(@PathVariable String applicationId, HttpSession httpSession) 
+          throws IOException{
+    if (applicationData == null || applicationData.getId() == null){
+      log.info("Application is empty or the applicationId is null when client attempts to download pdfs zip file.");
+      return createRootPageResponse();
+    }
+    MDC.put("applicationId", applicationData.getId());
+    MDC.put("sessionId", httpSession.getId());
+    log.info("Client with session: " + httpSession.getId() + " Downloading application with id: " + applicationData.getId());
+    Application application = applicationRepository.find(applicationId);  
+    List<ApplicationFile> applicationFiles = new ArrayList<>();
+    ApplicationData applicationDatas = application.getApplicationData();
```

Comment:
Not sure why we have an applicationData and an applicationDatas variable which both point to the same data structure here?